src/index.html

```diff
@@ -4,10 +4,10 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Leader Board</title>
 </head>
 <body>
-
+  <img src="images/header.png" alt="Header">
```

Comment:
- [ ] [OPTIONAL] I would suggest removing this since we don't need it, It might cause confusion to the user thinking he can click a button in the image while he couldn't. Your project is fine with just the to-do list if you wish not to implement a header/footer. If your choose to keep it I would recommend making it as a background image at least since it's not a part of the content